README.md

```diff
@@ -2,6 +2,31 @@
 
 This is The Jitsi Handbook. Your one-stop-shop for Jitsi documentation. It's powered by [Docusaurus](https://docusaurus.io/).
 
+## Install dependencies
+
+If you use **Windows**, you need install ```gifsicle```:
+
+```js 
+npm install -g gifsicle
+```
+alternative command
```

Comment:
Add `:` behind `alternative command`.

Reply:
> Add `:` behind `alternative command`.

I don't know why my submission of this PR is not tracking, I re submitted a [PR,](https://github.com/jitsi/handbook/pull/70) please review